hue/hue.sh

```diff
@@ -11,9 +11,14 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+# This script installs the latest version of HUE on a master node within a Google Cloud Dataproc cluster.
 
 set -x -e
 
+function err() {
+  echo "[$(date +'%Y-%m-%dT%H:%M:%S%z')]: $@" >&2
+}
```

Comment:
@bsidhom Suggested to avoid using ```exit``` in order to control script flow. Please check conversation:  https://github.com/GoogleCloudPlatform/dataproc-initialization-actions/pull/188/files/c8e43dce993889f7bfa9748df7666ae6801651c3

Reply:
According to that conversation, this needs an `exit 1` at the end

Reply:
Fixed.